app/assets/stylesheets/administrate/library/_variables.scss

```diff
@@ -15,6 +15,7 @@ $heading-line-height: 1.2 !default;
 $base-border-radius: 4px !default;
 $base-spacing: $base-line-height * 1em !default;
 $small-spacing: $base-spacing / 2 !default;
+$top-space: 0.39em !default;
```

Comment:
I feel that `top-space` it a bit generic a name. Should this be something else? I guess it'll depend on whether `cell-label` needs to be altered (see my comment above).